README.md

```diff
@@ -149,3 +149,20 @@ Writing short FAQs is quite easy and there are only few things to consider.
 2. To separate parts of your article with a thin line, use `<hr/>` tags. 
 3. To insert images, use the `<img/>` tag inside of a `<figure>` tag. Additionally, each image needs a caption (`<figcaption>`).
 4. Please keep your sentences as shorts as possible. This makes it easier to follow you and your instructions.
+
+#### Usage of WEB-API
```

Comment:
```suggestion

#### Usage of web API

```

Reply:
Please add a note that this API is for NewPipe's own purposes only and should not be used by other people. They can set up their own instance if they want to.

Reply:
> Please add a note that this API is for NewPipe's own purposes only and should not be used by other people. They can set up their own instance if they want to.

I think this note belongs in the API repo.

Reply:
You could say the same about the entire section.

The question is, will people actually read the API *repo* documentation if they just want to use the API themselves, now that they discovered it? I think a short note wouldn't hurt. We can add one to the API repo, too, of course.

Reply:
I added:
> If you want to use the API for other purposes than running or developing our website, please host it yourself.